rclcpp/include/rclcpp/any_subscription_callback.hpp

```diff
@@ -35,17 +35,23 @@ struct AnySubscriptionCallback
   using SharedPtrCallback = std::function<void(const std::shared_ptr<MessageT> &)>;
   using SharedPtrWithInfoCallback =
       std::function<void(const std::shared_ptr<MessageT> &, const rmw_message_info_t &)>;
+  using ConstSharedPtrCallback = std::function<void(const std::shared_ptr<const MessageT> &)>;
```

Comment:
ok, that wasn't as bad as I thought, I just changed a few places where the `unique_ptr` copy constructor was being used. I'll restart the CI jobs.